assets/src/edit-story/components/keyboardShortcutsMenu/shortcutLabel.js

```diff
@@ -31,18 +31,25 @@ function ShortCutLabel({ keys, alignment = 'center', ...rest }) {
     () =>
       keys.map((key, index) =>
         key.label ? (
-          <ShortcutKeyLabel key={index}>{key.label}</ShortcutKeyLabel>
+          <ShortcutKeyLabel key={key.label}>{key.label}</ShortcutKeyLabel>
         ) : (
-          <ShortcutKey
-            key={index}
-            aria-label={key.title}
-            title={key.title}
-            keySize={
-              (key.symbol || key).length > 1 ? KEY_SIZE.LARGE : KEY_SIZE.NORMAL
-            }
-          >
-            {key.symbol || key}
-          </ShortcutKey>
+          <>
+            {/* Disable reason: some keys consist of just a word, e.g. "or" -- there is nothing else than index unique in these. */}
+            {/* eslint-disable react/no-array-index-key */}
```

Comment:
Maybe we could use `key.symbol || key` as for the label? That ought to be unique within the same shortcut, no?

Reply:
Sometimes `key` doesn't even exist. I found one combination where the value was always unique -- expect for when the only value available is just `or` -- there is no way to make that unique unfortunately and the value "or" is used twice.